### PR TITLE
Typos in StudentProfileAttributesTest, CoursesLogicTest and InstructorCoursesPageDataTest #6991

### LINKED ISSUE #6991: Typos in StudentProfileAttributesTest, CoursesLogicTest and InstructorCoursesPageDataTest
StudentProfileAttributesTest.java
Invalidty is not a word according to provided dictionary
``` java
testGetInvalidtyInfoForValidProfileWithEmptyValues();
```

CoursesLogicTest.java
Cousre is not a word according to provided dictionary
``` java
        CourseAttributes notSampleCousre = new CourseAttributes("course.id", "not sample course", "UTC");
```

InstructorCoursesPageDataTest.java
Withour is not a word according to provided dictionary
``` java
InstructorCoursesPageData pageData = new InstructorCoursesPageData(instructorAccountWithourCourses);
```

### PR DESCRIPTION
**Pull request by Malkone**
Typos in StudentProfileAttributesTest, CoursesLogicTest and InstructorCoursesPageDataTest #6991
Fixes #6991

Corrected all typos described in issue:
'Invalidity' changed to 'Failure'
'Cousre' changed to 'Course'
'Withour' changed to 'Without'

### CODE REVIEW

File: src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ private void testGetInvalidityInfoForValidProfileWithValues() {
         assertEquals(new ArrayList<String>(), validProfile.getInvalidityInfo());
     }
 
-    private void testGetInvalidtyInfoForValidProfileWithEmptyValues() {
+    private void testGetFailureInfoForValidProfileWithEmptyValues() {
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java]: I believe the problem here is that invalidity is mispelt so it should be changed to `testGetInvalidityInfoForValidProfileWithEmptyValues`.

File: src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java [hunk 2]
```diff
@@ -100,7 +100,7 @@ private void testGetInvalidtyInfoForValidProfileWithEmptyValues() {
         assertEquals(new ArrayList<String>(), validProfile.getInvalidityInfo());
     }
 
-    private void testInvalidityInfoForInvalidProfile() throws Exception {
+    private void testFailureInfoForInvalidProfile() throws Exception {
```
Comment (leeyimin) [inline src/test/java/teammates/test/cases/datatransfer/StudentProfileAttributesTest.java]: Same here.

**Comment by xpdavid** [comment]
Hi @Malkone, I see you have already one d.FirstTimers fix ongoing (#6978). Please fix issues with the d.Contributors label instead and leave this for the new contributors :)

**Comment by Malkone** [comment]
Agree :) Thank you!

**Pull request closed without merging**